create_infotexts.py

```diff
@@ -374,15 +375,44 @@ def process_depicted_place(self, place_string, places_mapping):
                 place_as_wikitext = place_string
 
             # Don't forget to add the commons categories, even though only wikidata is used in depicted people field
-                if places_mapping[place_string].get('commonscat'):
-                    self.content_cats.append(places_mapping[place_string]["commonscat"])
+            if places_mapping[place_string].get('commonscat') and \
+            places_mapping[place_string].get('commonscat') != "-" and \
+            places_mapping[place_string].get('commonscat') not in self.content_cats:
```

Comment:
I wouldn't worry about duplicates here but instead change [line 226](https://github.com/mattiasostmar/SMVK-Cypern_2017-01/blob/2d1e99df5cbda4f2b8ac4a29ae9115c3f647fc2b/create_infotexts.py#L226)
from:
`img_info["cats"] = img.content_cats`
to
`img_info["cats"] = list(set(img.content_cats))`

and similar for ´meta_cats`.